deps.js

```diff
@@ -0,0 +1 @@
+console.info(require('child_process').execSync('npm ci --ignore-scripts', { cwd: __dirname }).toString());
```

Comment:
This handles package pre-installation, instead of storing `node_modules/` in it. When `node_modules/` contain binaries, they won't run anyway

Reply:
Is this recommended way to remove node_modules from repository?

The doc seems recommending the use of ncc

> When node_modules/ contain binaries, they won't run anyway

I don't think our dependency includes any binary

Reply:
> I don't think our dependency includes any binary
Yeah no binaries in our deps atm

> The doc seems recommending the use of ncc
I'd say safer to use a docker images for those scenarios, as ncc still builds a platform and architecture-specific binary

Reply:
> Is this recommended way to remove node_modules from repository?

Nope, technically, there's no recommended way – this one's straightforward though, and still guarantees the result. 
The docs kinda suggest just ignoring it and keeping `node_modules/` in the repo but I can name, like, a hundred reasons why it's _totally wrong_ – I'm 100% sure this is just a bloody hack GitHub devs had to suggest to get away with